docs/_pages/performance.md

```diff
@@ -155,3 +155,9 @@ In addition to a fast allocator, all frequently used structures in Luau have bee
 While the best performing code in Luau spends most of the time in the interpreter, performance of the standard library functions is critical to some applications. In addition to specializing many small and simple functions using the builtin call mechanism, we spend extra care on optimizing all library functions and providing additional functions beyond the Lua standard library that help achieve good performance with idiomatic code.
 
 For example, functions like `insert`, `remove` and `move` from the `table` library have been tuned for performance on array-like tables, achieving 3x and more performance compared to un-tuned versions, and Luau provides functions like `table.create` and `table.find` to achieve further speedup when applicable. We also use a carefully tuned dynamic string buffer implementation for internal `string` library to reduce garbage created during string manipulation.
+
+## Improved garbage collector pacing
+
+Luau uses an incremental garbage collector which does a little bit of work every so often, and at no point does it stop the world to go through the entire heap. The runtime will make sure that the collector runs interspersed with the program execution as the program allocates additional memory, which is known as "garbage collection assists", and can also run in response to explicit garbage collection invocation. In interactive environments such as video game engines it's possible, and even desireable, to request garbage collection every frame to make sure assists are minimized, since that allows scheduling the garbage collection to run concurrently with other engine processing that doesn't involve script execution.
```

Comment:
"GC assists" isn't terminology I'm familiar with. Is this the same as "barriers"?

Reply:
Nope - it's bespoke. It literally means that the garbage collection step will "assist" the collection process as the program runs.

Reply:
Oh, I recall now, the language is borrowed from some Go's design papers (their collector is concurrent but also needs to have the mutator to stop and assist the collector once in a while to maintain correct pacing, hence the term). I don't know if they came up with that themselves or if it's borrowed from the literature.